# Single-cell and bulk RNASeq analysis
Example analysis scripts with Seurat, Monocle, and other R packages for normalization, scaling, dimensionality reduction techniques (tSNE, PCA), differential gene expression, and visualization. For scRNASeq from 10X Genomics Chromium platform, I use cellranger demultiplexing, and for the ddSeq platform, I use Illumina BaseSpace automated demultiplexing. Both these programs then run a customized STAR alignment. Seurat is my package of choice for single-cell data. For bulk RNASeq, I use a manual STAR-htseq-DESeq2 pipeline but may switch to limma instead of DESeq2. For visualization purposes, I use the R packages' built-in functions or use custom ggplot2 and occasionally base R graphics functions.
